### PR TITLE
SQLite $select->combine() support

### PR DESCRIPTION
**Pull request by mwillbanks**
SQLite does not allow parentheses in a union.  You can find this in the
documentation on SQLite located at:
https://www.sqlite.org/lang_select.html as a secondary, you may look to
a tutorial on unions in SQLite at:
http://www.sqlitetutorial.net/sqlite-union/

This resolves issue #75 - SQLite error on UNION.

### CODE REVIEW

File: test/Sql/Platform/Sqlite/SelectDecoratorTest.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    /**
+     * @testdox integration test: Testing SelectDecorator will use Select an internal state to prepare a proper limit/offset sql statement
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::getSqlString
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processLimit
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processOffset
+     * @dataProvider dataProvider
+     */
+    public function testGetSqlString(Select $select, $ignore, $alsoIgnore, $expectedSql)
```
Comment (weierophinney) [inline test/Sql/Platform/Sqlite/SelectDecoratorTest.php]: Can you give a more descriptive name, please? What is the behavior you're testing? E.g., "testGetSqlStringCorrectlyGeneratesUnionSyntaxFromCombine".

File: test/Sql/Platform/Sqlite/SqliteTest.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+use Zend\Db\Sql\Platform\Sqlite\Sqlite;
+
+class SqliteTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @testdox unit test / object test: Test Sqlite object has Select proxy
+     * @covers Zend\Db\Sql\Platform\Sqlite\Sqlite::__construct
+     */
+    public function testConstruct()
```
Comment (weierophinney) [inline test/Sql/Platform/Sqlite/SqliteTest.php]: Better name, please: "testConstructorRegistersSqliteDecorator"

File: test/Sql/Platform/Sqlite/SelectDecoratorTest.php (hunk 1 above)
Comment (mwillbanks) [inline test/Sql/Platform/Sqlite/SelectDecoratorTest.php]: I can, however, the existing tests on the platform all utilized this exact behavior, see the oracle and mysql tests for example.

File: test/Sql/Platform/Sqlite/SqliteTest.php (hunk 2 above)
Comment (mwillbanks) [inline test/Sql/Platform/Sqlite/SqliteTest.php]: I can, same as previous comment, i kept it consistent with the other Platform tests.

**Pull request merged**